configure.ac

```diff
@@ -93,6 +93,22 @@ case "${host_os}" in
 	;;
 	darwin*)
 		ISDARWIN=1
+
+		AC_CHECK_PROG([BREW],brew, brew)
+		if test x$BREW = xbrew; then
+			dnl add default homebrew paths
+
+			icu4c_prefix=$($BREW --prefix icu4c)
+			export PKG_CONFIG_PATH="$icu4c_prefix/lib/pkgconfig:$PKG_CONFIG_PATH"
+
+			python3_prefix=$(dirname "$(find "$($BREW --prefix python3)/" -name python3.pc)")
+			export PKG_CONFIG_PATH="$python3_prefix:$PKG_CONFIG_PATH"
+
+			if test -z "$USE_SYSTEM_OPENSSL" -o "$USE_SYSTEM_OPENSSL" = "false"; then
```

Comment:
IMHO this does not depend on travis.
If the user want to compile ZNC using the system openssl `USE_SYSTEM_OPENSSL` can be used in the same way.
Maybe I should add a description line to the configure script so this is documented somewhere?